internal/translation/maintenancewindow/conversion_test.go

```diff
@@ -0,0 +1,30 @@
+package maintenancewindow
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	fuzz "github.com/google/gofuzz"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRoundtrip_MaintenanceWindow(t *testing.T) {
```

Comment:
FYI gofuzz [supports integration with dvyukovgo-fuzz](https://github.com/google/gofuzz?tab=readme-ov-file#dvyukovgo-fuzz-integration) which in turns means we might be able to make our fuzz tests official go fuzz tests, with the language support that this brings.
No need to change for this one, but maybe we could explore it going forward.

Reply:
that's indeed a nice integration. (non blocking for this PR)